kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,65 @@
+### Domain lifecycle sample scripts
+The WebLogic Server Kubernetes Operator project provides a set of sample scripts to shut down or start a specific managed-server, cluster or the entire domain. These can be helpful when scripting the lifecycle of a WebLogic Server Domain.
```

Comment:
Please add a link back to the lifecycle documenation here.  Note that because this is not underr 'docs-source/content' the link will need to be a full URL (github.io/...) or a relatively link (../..).  Often, both styles of link are helpful - for example:

{code}
./create-weblogic-domain/model-in-image/README.md:  - [source](../../../../../docs-source/content/samples/simple/domains/model-in-image/_index.md)
./create-weblogic-domain/model-in-image/README.md:  - [latest public](https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/domains/model-in-image)
{code}

Reply:
LGTM